mysql-test/suite/galera/t/mdev-30653.test

```diff
@@ -0,0 +1,49 @@
+--source include/galera_cluster.inc
+
+create table t1 (id serial, val int) engine=innodb;
+create table t2 (id serial, val int) engine=aria;
+ 
+insert into t1 values(1, 23);
+insert into t2 values(2, 42);
+call mtr.add_suppression("WSREP: Replication of non-transactional engines are experimental. Storage engine Aria for table 'test'.'t2' is not supported in Galera");
+
+begin;
+update t1 set val=24 where id=1;
+update t2 set val=41 where id=2;
+--error ER_ERROR_DURING_COMMIT
+commit;
+
+select * from t1;
+select * from t2;
+
+--connection node_2
+select * from t1;
+select * from t2;
+
+--connection node_1
+drop table t1, t2;
+
+# case 2
+SET GLOBAL wsrep_mode=REPLICATE_ARIA;
+create table t1 (id serial, val int) engine=innodb;
+create table t2 (id serial, val int) engine=aria;
+ 
+insert into t1 values(1, 23);
+insert into t2 values(2, 42);
+ 
+begin;
+update t1 set val=24 where id=1;
+--error ER_ERROR_DURING_COMMIT
+update t2 set val=41 where id=2;
+commit;
+
+select * from t1;
+select * from t2;
+
+--connection node_2
+select * from t1;
+select * from t2;
+
+--connection node_1
+drop table t1, t2;
+SET GLOBAL wsrep_mode=DEFAULT;
```

Comment:
for test completeness, would be good to have one more test scenario, where a plain UPDATE to ARIA table works and is replicated under REPLICATE_ARIA mode

Reply:
Added.